src/models/modules/utils/create_act.py

```diff
@@ -0,0 +1,137 @@
+"""TorchOK Activation Factory.
```

Comment:
Do we really need this file, if yes replace it in briks path and add documentation.